data_sources/commodities.py

```diff
@@ -0,0 +1,93 @@
+import csv
+import requests
+import datetime
+import logging
+
+from data_sources.data_source import DataSource
+from training.cleaning_pipeline import CleaningPipeline
+
+logger = logging.getLogger(__name__)
+
+
+class CommoditiesDataSource(DataSource):
+    HOST = "https://reporting.trade-tariff.service.gov.uk"
+    PATH = "/:service/reporting/:year/:month/:day/commodities_:service_:year_:month_:day.csv"
+    DEFAULT_SERVICE = "uk"
+
+    def __init__(
+        self,
+        service: str = DEFAULT_SERVICE,
+        cleaning_pipeline: CleaningPipeline | None = None,
+        authoritative: bool = False,
+        creates_codes: bool = False,
+        multiplier: int = 1,
+    ):
+        url = f"{self.HOST}{self.PATH}"
+        today = datetime.datetime.now()
+        year = str(today.year)
+        month = str(today.month).rjust(2, "0")
+        day = str(today.day).rjust(2, "0")
+
+        url = (
```

Comment:
I don't like reassigning the variable, especially with a different type, 
but it's just personal, nothing wrong :smile_cat:

Reply:
Its the same type, though

Reply:
We could create a new concept (unsubbed_url) or something